modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -3072,7 +3072,7 @@ void GDScriptAnalyzer::reduce_call(GDScriptParser::CallNode *p_call, bool p_is_a
 				push_error(vformat(R"*(Cannot get return value of call to "%s()" because it returns "void".)*", function_name), p_call);
 			}
 
-			if (all_is_constant && Variant::get_utility_function_type(function_name) == Variant::UTILITY_FUNC_TYPE_MATH) {
+			if (all_is_constant && (Variant::get_utility_function_type(function_name) == Variant::UTILITY_FUNC_TYPE_MATH || function_name == "str")) {
```

Comment:
I think we should move `str` (and probably a few other functions) to a new category, or add a `const`/`pure` flag like `GDScriptUtilityFunctions` does, rather than hardcoding individual functions.

Reply:
Sure. That would be more practical.